bin/audit-svg-colors.js

```diff
@@ -257,16 +290,14 @@ function findClosestColor( value, targetValues ) {
 }
 
 function findPaletteColorName( value ) {
-	let name;
-
 	// Iterating from right to make sure color name aliases aren’t caught
-	_.forInRight( PALETTE.colors, ( paletteValue, paletteColorName ) => {
-		if ( paletteValue === value ) {
-			name = paletteColorName;
-			return false;
-		}
-	} );
-
+	const name = PALETTE?.colors
+		? Object.keys( PALETTE.colors )
+				.reverse()
+				.find( ( paletteColorName ) => {
```

Comment:
Instead of calling `.reverse().find()` we could just use [`Array.prototype.findLast()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findLast).